extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/taskadapter/TaskAdapter.java

```diff
@@ -29,6 +29,6 @@ public interface TaskAdapter
 
   Job fromTask(Task task) throws IOException;
 
-  Task toTask(Job from) throws IOException;
+  String getTaskId(Job from) throws IOException;
```

Comment:
Seems implementations for this method are the same in diff subclasses, can make it default method implementation.